lib/panoptes/client/subjects.rb

```diff
@@ -16,6 +16,19 @@ def subjects(subject_set_id: nil, workflow_id: nil)
         response.fetch("subjects")
       end
 
+      # Determine whether a particular subject is accessible to a particular project
+      #
+      # @param subject_id [Integer]
+      # @param project_id [Integer]
+      # @return nil or the subject
+      def subject_in_project?(subject_id, project_id)
+        response = panoptes.get("/subjects/#{subject_id}?project_id=#{project_id}")
+        return response['subjects'][0] if response&.fetch('subjects')&.count > 0
```

Comment:
If we're expecting this to contain zero or one elements in the array, I tend to prefer to raise an exception if Panoptes somehow ends up returning more than one element. Should never happen, but if it does it will be hell to debug without that exception.

Reply:
that's a good point, better to think ahead here